hypervisor/arch/x86/configs/nuc7i7dnb.config

```diff
@@ -1,5 +1,5 @@
 # Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib)
 CONFIG_BOARD="nuc7i7dnb"
 CONFIG_SERIAL_LEGACY=y
-CONFIG_HV_RAM_START=0x60000000
+CONFIG_HV_RAM_START=0x41000000
```

Comment:
@liuwei987  Could you add a description why we modify HV RAM START address in this PR?